models/models.gen.go

```diff
@@ -50,6 +50,8 @@ type EventLog struct {
 	Address          Address  `json:"address"`
 	Data             string   `json:"data"`
 	LogIndex         Counter  `json:"logIndex"`
+	Name             string   `json:"name"`
+	Params           []string `json:"params"`
```

Comment:
I think its better to have a separate struct for this. And add it here as optional field